scylla/src/transport/session.rs

```diff
@@ -64,6 +65,7 @@ use crate::transport::cluster::{Cluster, ClusterData, ClusterNeatDebug};
 use crate::transport::connection::{Connection, ConnectionConfig, VerifiedKeyspaceName};
 use crate::transport::connection_pool::PoolConfig;
 use crate::transport::host_filter::HostFilter;
+#[allow(deprecated)]
 use crate::transport::iterator::{LegacyRowIterator, PreparedIteratorConfig};
```

Comment:
Is it ok? I'm looking at `PreparedIteratorConfig`. The analogous import:
```rust
use crate::{LegacyQueryResult, QueryResult};
```

was decoupled into:
```rust
#[allow(deprecated)]
use crate::LegacyQueryResult;
use crate::QueryResult;
```

Reply:
It does not hurt. We allow deprecations in a bit more places than we would like, but it's benign.

Reply:
I would left this as-is.

Reply:
Ok, then ignore my remaining comments